### DISCUSSION TITLE
Seed Labs 1.0 - Mobility

### DISCUSSION THREAD

**Discussion by parthian88** [seed-labs/seed-labs#147] [Q&A]
Professor Du,

I know you are beginning to phase it out, but I am still trying to do it on an Intel Mac. I can get the older Ubuntu VM to start but the Android VM just shows a black screen with a flashing cursor in the top left corner.  never goes beyond that and wwas wondering if you had any suggestions.

I have also tried downgrading virtual box to 6.x but that gives an error when i try to start:

<html><body><p>Failed to open a session for the virtual machine <b>androidvm7</b>.</p><qt><p>The virtual machine <b><nobr>'androidvm7'</nobr></b> has terminated unexpectedly during startup because of signal 6.</p>
Result Code: | NS_ERROR_FAILURE (0x80004005)
-- | --
Component: | MachineWrap
Interface: | IMachine {85632c68-b5bb-4316-a900-5eb28d3413df}

<br></body></html>

Thanks, in advance for insight you may have.

Sajjad

**Comment by kevin-w-du** [comment]
I haven't touched the Android VM for many years, I will be surprised if it still works. I simply don't have the man power to maintain this VM. In this coming year, if I get enough resources, I will try to rebuild this VM and revive the android security labs.
